site/_data/site.json

```diff
@@ -12,7 +12,8 @@
     "ko",
     "zn",
```

Comment:
Should this zn be changed to `zh`? I'm not sure if `zn` is a locale?

Reply:
Yes I think so. Searching "zn" on various locale sites yields nothing. Also we don't have a locale directory of "zn", so I think it would be safe to change "zn" to "zh".